packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs

```diff
@@ -59,6 +59,7 @@ export class <%= controllerClassName %> {
       content: {
         'application/json': {
           schema: getModelSchemaRef(<%= targetModelClassName %>, {
+            title: 'New <%= targetModelClassName %>In<%= sourceModelClassName %>',
```

Comment:
Would it be possible to add a test for this line?

Reply:
Good catch! AFAICT, we don't have any test coverage for `controller-relation-template-has-many.ts.ejs` yet. I'll open a new pull request where I'll introduce the necessary testing infrastructure and also add a test for this line.

Reply:
Eventually I found those tests, see e.g. [packages/cli/test/integration/generators/hasmany.relation.integration.js](https://github.com/strongloop/loopback-next/blob/8f49a4552f8a4792b2e45b631e6c7422c718e7c3/packages/cli/test/integration/generators/hasmany.relation.integration.js).

I found the existing tests for `lb4 relation` rather messy. Before I add new assertions or test cases, I'd like to refactor & clean up the test suite. See https://github.com/strongloop/loopback-next/pull/3748

I'll add the new test(s) once the refactorings are landed.

Reply:
I opened a PR to add the missing test, see https://github.com/strongloop/loopback-next/pull/3754